projboard/templates/myArticles/my_articles.html

```diff
@@ -0,0 +1,74 @@
+{% extends "base.html" %}
+{% load static %}
+{% block content %}
+<!doctype html>
+<html lang="en">
+
+<head>
+  <!-- Required meta tags -->
+  <meta charset="utf-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
```

Comment:
What is found from line 9 to 11, is found in base.html and in my opinion it is not necessary here either

Reply:
done at [3e5e181](https://github.com/redhat-beyond/Smarticle/pull/87/commits/3e5e181b5adfe2ea1d8c147e8ee20d57c25a89a9)